pymc/sampling.py

```diff
@@ -1557,21 +1559,25 @@ def sample_posterior_predictive(
         or xarray.Dataset (eg. InferenceData.posterior or InferenceData.prior)
     samples : int
         Number of posterior predictive samples to generate. Defaults to one posterior predictive
-        sample per posterior sample, that is, the number of draws times the number of chains. It
-        is not recommended to modify this value; when modified, some chains may not be represented
-        in the posterior predictive sample.
+        sample per posterior sample, that is, the number of draws times the number of chains.
+
+        It is not recommended to modify this value; when modified, some chains may not be
+        represented in the posterior predictive sample. Instead, in cases when generating
+        posterior predictive samples is too expensive to do it once per posterior sample,
+        the recommended approach is to thin the ``trace`` argument
+        before passing it to ``sample_posterior_predictive``. In such cases it
```

Comment:
A short example would be great. I know it's a one-liner, but I can't remember how...